1. objects/1.md

```diff
@@ -2,72 +2,119 @@
 
 ```js
 // your code goes here
+var myPenguin = {
+
+	name : "whiteblack",
+	origin : "whiteblack the penguin see the world",
+	author : "H.A. Rey and Margret Rey"
+
+}
 ```
 
 2. Use console.log() to print the penguin's name to the console as part of a welcome message, like "Hello, I'm a penguin and my name is [NAME HERE]!"
 
 ```js
 // your code goes here
+
+console.log(`Hello, I'm a penguin and my name is ${myPenguin.name}`)
 ```
 
 3. Write another line of code that adds a new property to your penguin called canFly and set it to false.
 
 ```js
 // your code goes here
+var myPenguin = {
+
+	name : "whiteblack",
+	origin : "whiteblack the penguin see the world",
+	author : "H.A. Rey and Margret Rey",
+	canFly : "false"
+
+}
 ```
 
 4. Add a method to your penguin called chirp that prints to the console: "CHIRP CHIRP! Is this what penguins sound like?"
 
 ```js
 // your code goes here
+myPenguin.chirp = function() {
+	alert("CHIRP CHIRP! Is this what penguins sound like");
+}
+
+myPenguin.chirp();
+
 ```
 
 6. Next, call your penguin's sayHello() method and make sure that it works!
 
 ```js
 // your code goes here
+myPenguin.sayHello = function() {
+	alert("Hello")
+}
+
+myPenguin.sayHello();
 ```
 
 7. Without modifying any of your previous code, change the penguin's name to "Penguin McPenguinFace" and then call your penguin's sayHello() function one more time to make sure it still works.
 
 ```js
 // your code goes here
+myPenguin.name = "Penguin McPenguinFace";
+
+myPenguin.sayHello();
 ```
 
 8. Write another method called fly, and inside that method, use an if / else statement to print "I can fly!" to the console if your penguin's canFly property is true, or "No flying for me!" if its canFly property is false.
 
 ```js
 // your code goes here
+myPenguin.fly = function(canfly) {
+	if (canfly == true){alert("I can fly!")}
+	else (canfly == false){alert("No flying for me!")}
```

Comment:
`else` doesn't take a condition. `if` is executed when the expression is true and else is executed when the expression in `false`.

The code should be like this:
```
if (canfly == true){
  alert("I can fly!");
} else {
  alert("No flying for me!");
}
```